### PR TITLE
Add three additional instances to support HA lab

### PR DESCRIPTION
**Pull request by hikhvar**

### CODE REVIEW

File: modules/student_workspace/main.tf [hunk 1]
```diff
@@ -40,8 +40,35 @@ module "node0" {
...
+  name            = "proxy"
+  public_ssh_keys = "${tls_private_key.ssh_key.*.public_key_openssh}"
+  network         = "${var.network}"
+  machine_type    = "${var.machine_type}"
+}
+
+module "second_master" {
+  source          = "../student_node"
+  students        = "${var.students}"
+  name            = "second-master"
```
Comment (johscheuer) [inline modules/student_workspace/main.tf]: Just for consistency shouldn't be the first master the `first-master` also I would use names like `master01`, `master02` (or are the nodes named like this in the exercise?

File: modules/student_workspace/main.tf [hunk 2]
```diff
@@ -40,8 +40,35 @@ module "node0" {
...
+  name            = "third-master"
+  public_ssh_keys = "${tls_private_key.ssh_key.*.public_key_openssh}"
+  network         = "${var.network}"
+  machine_type    = "${var.machine_type}"
+}
+
 resource "local_file" "public_ips" {
   count    = "${length(var.students)}"
-  content  = "master: ${module.master.public_ip[count.index]}\nnode: ${module.node0.public_ip[count.index]}\n"
+  content  = "master: ${module.master.public_ip[count.index]}\nnode: ${module.node0.public_ip[count.index]}\nproxy: ${module.proxy.public_ip[count.index]}\nsecond-master: ${module.second_master.public_ip[count.index]}\nthird-master: ${module.third_master.public_ip[count.index]}\n"
```
Comment (johscheuer) [inline modules/student_workspace/main.tf]: Same here (naming)

File: modules/student_workspace/main.tf (hunk 1 above)
Comment (hikhvar) [reply modules/student_workspace/main.tf]: they are named `second-master` and `third-master` in the exercise. I don't like that convention either. But I sticked to the naming in the labs.

Comment (johscheuer) [reply modules/student_workspace/main.tf]: Then it's fine for me

**Pull request merged**